gson/pom.xml

```diff
@@ -10,6 +10,13 @@
   <artifactId>gson</artifactId>
   <name>Gson</name>
 
+  <licenses>
+    <license>
+      <name>Apache 2.0</name>
```

Comment:
The [Maven POM Reference](https://maven.apache.org/pom.html#licenses) says:
> Using an [SPDX identifier](https://spdx.org/licenses/) as the license name is recommended.

The identifier for Apache 2.0 seems to be [`Apache-2.0`](https://spdx.org/licenses/Apache-2.0.html).
(Though even the Maven POM Reference example does not follow that advice; not sure how important it is.)

---

Note that I am not a maintainer of this repository; this comment is only a suggestion.

Reply:
I took the liberty of making that change. Thanks @friendoye and @Marcono1234!